Makefile

```diff
@@ -1,25 +1,42 @@
 LOG_LEVEL ?= spin=trace
 CERT_NAME ?= local
 SPIN_DOC_NAME ?= new-doc.md
+export PATH := target/debug:target/release:$(HOME)/.cargo/bin:$(PATH)
```

Comment:
Shouldn't this use `CARGO_TARGET_DIR` or the `target` directory if `CARGO_TARGET_DIR` is not set?

Reply:
when using cargo_target_dir, we had issues with where `.wasm` files would be saved when running `spin build`.

Reply:
I guess I'm confused what this line is for?

Reply:
this is for local laptop scenario. e.g. if you run `make test-kv`, it runs the test directly on the laptop (instead of a container), so idea is to use spin from either of the directory (in that order)

Reply:
Hi @rylev, I want to make sure I have answered all your questions before I mark this comment as resolved. pls let me know if you have more questions.

Reply:
Ok this is fine for now. I just really wish all the tests could rely on a spin binary being in one canonical location and then we could have logic to decide how to copy a binary from a number of different possible locations to that canonical location. Right now, it feels a bit of like a house of cards as to which spin binary will actually be used.

That doesn't need to delay merging this anymore.